icypi/PIsense.py

```diff
@@ -0,0 +1,9 @@
+from sense_hat import sense_hat
+
+sense = SenseHat()
+sense.clear
+
+temp = sense.get_temperature()
+
+
+print(temp)
```

Comment:
Awesome!

Now attempt #3 to execute this script in Python using the Parmiko package to handle your ssh log in and script execution for you...